doc/releases/changelog-dev.md

```diff
@@ -70,6 +70,22 @@
 
 <h3>Deprecations 👋</h3>
 
+* The `expand_fn` argument in `qml.execute` has been deprecated. 
+  Instead, please create a `qml.transforms.core.TransformProgram` with the desired preprocessing and pass it to the `transform_program` argument of `qml.execute`.
+  [(#5984)](https://github.com/PennyLaneAI/pennylane/pull/5984)
+
+* The `max_expansion` argument in `qml.execute` has been deprecated. 
+  Instead, please use `qml.devices.preprocess.decompose` with the desired expansion level, add it to a `TransformProgram` and pass it to the `transform_program` argument of `qml.execute`.
+  [(#5984)](https://github.com/PennyLaneAI/pennylane/pull/5984)
+
+* The `override_shots` argument in `qml.execute` is deprecated. 
+  Instead, please add the shots to the `QuantumTape`'s to be executed.
```

Comment:
Is the 's in `QuantumTape`'s intentional?

Reply:
Yes, I meant to add it to indicate the plural of `QuantumTape`